docs/examples/TC_Tracks_Intensity.py

```diff
@@ -103,7 +103,7 @@
         lonpt, latpt = m(xpt, ypt, inverse=True)
         xs1.append(lon)
         ys1.append(lat)  
-cs2=m.plot(xs1, ys1, linestyle='--',color='Red',label='UFSv1.0.0_15p2_hybedmf')
+cs2=m.plot(xs1, ys1, linestyle='--',color='Red',label='MRW_GFSv15p2')
```

Comment:
Should `_hybedmf` be removed or changed to `_satmedmfvdifq`?

Reply:
This is a demonstration only plot script. The input tc tracker file only points to the MRW_GFSv15p2 and is not specific to any sensitivity run we did. I think it should be okay to just list MRW_GFSv15p2, which is also consistent with the plot in the hurricane case page.